docs/Introduction/tutorials/advanced-tutorial.md

```diff
@@ -11,47 +11,64 @@ metadata:
++ [5. How do I deploy to testnet?](#5-how-do-i-deploy-to-testnet)
++ [6. Further Reading](#6-further-reading)
+
+# 1. How does the request and receive cycle work for API calls?
 
 The request and receive cycle describes how a smart contract requests data from an oracle and receives the response in a separate transaction. If you need a refresher, check out the [Basic Request Model](../architecture-request-model/).
 
-In the [Random Numbers tutorial](../intermediates-tutorial/), we request randomness from a VRF oracle, then await the response. The fulfilment function is already given to us from the `VRFConsumerBase` contract, so oracles already know where to send the response to. However, with API calls, our contract _defines_ which function it wants to receive the response to.
+In the [Random Numbers](../intermediates-tutorial/) tutorial, you requested randomness from a VRF oracle, then await the response. The fulfilment function is already given to us from the `VRFConsumerBase` contract, so oracles already know where to send the response to. However, with API calls, the contract itself *defines* which function it wants to receive the response to.
+
+Before creating any code, you should understand how Oracle jobs can get data on-chain.
 
-However, before we go into the implementation, let's first understand how Oracle jobs can get data on-chain.
+# 2. What are initiators?
 
-## 1b. Initiators
+[**Initiators**](../initiators/) are what start, or initiate, a job inside an Oracle. In the case of a Request and Receive job, the [RunLog](/docs/initiators/#runlog) initiator monitors the blockchain for a request from a smart contract. Once it catches a request, it initiates the job. This runs the adapters (both core and external) that the job is configured to run eventually returning the response to the requesting contract.
 
-[Initiators](../initiators/) are what kick off a job inside an Oracle. In the case of a Request and Receive job, the [RunLog](/docs/initiators/#runlog) initiator watches the blockchain for when a smart contract makes a request. Once it catches a request, it initiates the job. This runs the adapters (both core and external) that the job is configured to run, eventually returning the response to the contract that made the request.
+# 3. What are Adapters?
 
-## 1c. Core Adapters
+Each oracle job has a configured set of tasks it must complete when it is run. These tasks are defined by the [**Adapters**](../core-adapters/) they support. Adapters are split into two subcategories:
 
-Each oracle job has a configured set of tasks it needs to carry out when it is run. These tasks are defined by what [Adapters](../core-adapters/) they support. For example: if a job needs to make a GET request to an API, find a specific unsigned integer field in a JSON response, then submit that back to the requesting contract, it would need a job with the following Core Adapters:
-- [HttpGet](../core-adapters/#httpget) - Call the API
```

Comment:
swap these for their task counterparts

Reply:
Captured here: https://github.com/smartcontractkit/documentation/issues/500

We'll address this in a followup.